factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/shortcodes/icon.md

```diff
@@ -0,0 +1,1254 @@
++++
+title = "icon"
+description = "Display an icon."
++++
+
+Display an icon like `{{% icon cloud %}}`
+
+It uses :
+
+* glyphicons library (bootstrap).[more info here](http://getbootstrap.com/components/#glyphicons)
+* fontawesome library [more info here](http://fontawesome.io/)
+
+This `icon` shortcode will display an icon in your page. 
+
+## Usage
+
+| Parameter | Default | Description |
+|:--|:--|:--|
+| name | **required** | name of icon (see bellow) |
+| size | none | size of icon, medium, xx-small, x-small, small, large, x-large, xx-large, 11px, 2em, 20%.... |
+
+{{%alert info%}}
+**Tips :**
+setting only the name as argument works too : `{{</*icon film*/>}}` instead of `{{</*icon name="film"*/>}}`
+{{%/alert%}}
+
+# Demo
+	
+	{{</* icon name="film" size="large" */>}}
+
+{{<icon name="film" size="large">}}
+
+.
+
+# icons available
+
+- {{< icon name="asterisk" size="large" >}} asterisk
+- {{< icon name="plus" size="large" >}} plus
+- {{< icon name="euro" size="large" >}} euro
+- {{< icon name="eur" size="large" >}} eur
+- {{< icon name="minus" size="large" >}} minus
+- {{< icon name="cloud" size="large" >}} cloud
+- {{< icon name="envelope" size="large" >}} envelope
+- {{< icon name="pencil" size="large" >}} pencil
+- {{< icon name="glass" size="large" >}} glass
+- {{< icon name="music" size="large" >}} music
+- {{< icon name="search" size="large" >}} search
+- {{< icon name="heart" size="large" >}} heart
+- {{< icon name="star" size="large" >}} star
+- {{< icon name="star-empty" size="large" >}} star-empty
+- {{< icon name="user" size="large" >}} user
+- {{< icon name="film" size="large" >}} film
+- {{< icon name="th-large" size="large" >}} th-large
+- {{< icon name="th" size="large" >}} th
+- {{< icon name="th-list" size="large" >}} th-list
+- {{< icon name="ok" size="large" >}} ok
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)